### PR TITLE
Update user guide

### PR DESCRIPTION
**Pull request by djleong01**
Add initial draft of user guide up to current version.

Can expand upon FAQ section more. To be updated as we go along.

### CODE REVIEW

File: docs/UserGuide.md [hunk 1]
```diff
@@ -1,42 +1,306 @@
...
+```dtd
+java -jar tp.jar
+```
+5. Upon starting the application, you will be prompted to enter your PIN. The PIN is required to access the application.
+The app will prompt you to create your own PIN if it is your first time using the application.
+6. You can now start using LongAh! by entering commands into the command terminal.
+
+
+## Quick Command Reference
+| Command                | Parameters                                                                                            |
```
Comment (1simjustin) [inline docs/UserGuide.md:27]: Change to "Task" and "Command Expression"

File: docs/UserGuide.md [hunk 2]
```diff
@@ -1,42 +1,306 @@
...
+### Saving the data
+LongAh! data is saved in the hard disk automatically after any command that changes the data. There is no need to save manually.
+The file is also created automatically if it does not exist.
+
+### Editing the data file
+LongAh! data is saved as a TXT file in the hard disk. Advanced users are welcome to edit the data file directly, but please ensure that the data is in the correct format.
+The PIN TXT file contains the pin hash of each user's PIN for security purposes. It is not recommended to edit this file directly.
+
+
+## Command Format
```
Comment (1simjustin) [inline docs/UserGuide.md:78]: Can add expected output following sample usage also

Comment (1simjustin) [reply docs/UserGuide.md:78]: or rather separate the expected output from the command invokation

**Comment by djleong01** [comment]
Gonna put this up for review since I got the base template out, also left some stuff for you guys to fill in (eg expected outputs)

**Review by jing-xiang — APPROVED** [review]
LGTM

**Pull request merged**